dags/multipod/legacy.py

```diff
@@ -70,3 +87,49 @@
         docker_image=DOCKER_IMAGE[test_mode].value,
         test_owner=test_owner.JON_B,
     ).run()
+
+    # v4-8 1 slice TFLOPS test
+    gke_config.get_gke_config(
+        tpu_version=TpuVersion.V4,
+        tpu_cores=8,
+        tpu_zone=Zone.US_CENTRAL2_B.value,
+        time_out_in_min=60,
+        test_name=f"maxtext-perf-v4-8-1-slices-{test_mode.value}",
+        run_model_cmds=(
+            f"bash end_to_end/test_tflops.sh xlml {tflop_thresholds['v4-8']['1']} gs://maxtext-xlml gs://maxtext-xlml/dataset xlml-tflops-v4-8-1slice-{test_mode.value}",
+        ),
+        docker_image=DOCKER_IMAGE[test_mode].value,
+        test_owner=test_owner.PRIYANKA_G,
+    ).run()
+
+    # v4-16 1 and 2 slice TFLOPS test
+    for n_slice in [1, 2]:
+      gke_config.get_gke_config(
+        tpu_version=TpuVersion.V4,
+        tpu_cores=16,
+        num_slices=n_slice,
+        tpu_zone=Zone.US_CENTRAL2_B.value,
+        time_out_in_min=60,
+        test_name=f"maxtext-perf-v4-16-{n_slice}-slices-{test_mode.value}",
+        run_model_cmds=(
+            f"bash end_to_end/test_tflops.sh xlml {tflop_thresholds['v4-16'][str(n_slice)]} gs://maxtext-xlml gs://maxtext-xlml/dataset xlml-tflops-v4-16-{n_slice}slice-{test_mode.value}",
+        ),
+        cluster_name=ClusterName.V4_16_MULTISLICE_CLUSTER.value,
+        docker_image=DOCKER_IMAGE[test_mode].value,
+        test_owner=test_owner.PRIYANKA_G,
+      ).run()
+
+  # v4-8 2 slices checkpoint resharding test
+  gke_config.get_gke_config(
+        tpu_version=TpuVersion.V4,
+        tpu_cores=8,
+        num_slices=2,
+        tpu_zone=Zone.US_CENTRAL2_B.value,
+        time_out_in_min=60,
+        test_name=f"maxtext-checkpoint-reshard-v4-8-2-slices-{SetupMode.STABLE.value}",
+        run_model_cmds=(
+            f"bash end_to_end/test_checkpoint_resharding.sh xlml-checkpoint-resharding-v4-8-2slice-{SetupMode.STABLE.value} gs://maxtext-xlml gs://maxtext-xlml/dataset",
```

Comment:
Do we only want to run this test on stable?

Reply:
We've previously only had this running on stable. Since we have main checkpointing tests testing nightly and stable anyways, resharding can just be a stable sanity check, I believe.